geowebcache/core/src/test/java/org/geowebcache/layer/wms/WMSLayerTest.java

```diff
@@ -173,7 +174,7 @@ protected void makeRequest(TileResponseReceiver tileRespRecv, WMSLayer layer,
                 ByteArrayOutputStream output = new ByteArrayOutputStream();
                 try {
                     ImageIO.write(img, "PNG", output);
-                    ImageIO.write(img, "PNG", new java.io.File("/tmp/meta.png"));
+                    ImageIO.write(img, "PNG", new java.io.File(SystemUtils.JAVA_IO_TMPDIR + "/meta.png"));
```

Comment:
Or even better, ./target, the global temp tends to accumulate garbage otherwise (ok, in this case it's a single file, not a big deal, disregard if you like)

Reply:
Fixed.